modules/core/src/main/java/org/apache/synapse/core/axis2/Axis2SynapseEnvironment.java

```diff
@@ -217,6 +222,38 @@ public void setSeverDebugMode(ServerContextInformation contextInformation) {
         }
     }
 
+    /**
+     * this method is to set the unit test mode is enabled
```

Comment:
Shall we start comment with capital letter

Reply:
Done